lib/db.js

```diff
@@ -186,22 +193,40 @@ DB.prototype.get = function(domain, req) {
         return this._getSchema(tableName)
         .then(function(schema) {
             self.schemaCache[tableName] = schema;
-            return self._get(tableName, req, schema);
+            return self._get(tableName, req, schema, true);
         });
     } else {
         return P.try(function() {
-            return self._get(tableName, req, self.schemaCache[tableName]);
+            return self._get(tableName, req, self.schemaCache[tableName], true);
         });
     }
 };
 
+DB.prototype._createGetQuery = function(tableName, req, schema, includePreparedForDelete) {
+    var extracted = dbu.extractGetParams(tableName, req, schema, includePreparedForDelete);
+    var key = stringify(req);
```

Comment:
@d00rman it does. See for example https://github.com/wikimedia/restbase-mod-table-sqlite/pull/12/files#diff-3b58377c7047ab11edbff7ec5d8f283aR438